### PR TITLE
Fix Windows constexpr issue and '#ifdef' column_family_test in Release.

### PR DESCRIPTION
**Pull request by yuslepukhin**

### CODE REVIEW

File: utilities/write_batch_with_index/write_batch_with_index_internal.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ struct WriteBatchIndexEntry {
 
   // If this flag appears in the offset, it indicates a key that is smaller
   // than any other entry for the same column family
-  static const size_t kFlagMin = std::numeric_limits<size_t>::max();
+  static const size_t kFlagMin = UINT64_MAX;
```
Comment (siying) [inline utilities/write_batch_with_index/write_batch_with_index_internal.h]: kMaxUint64?

**Comment by yuslepukhin** [comment]
@siying What do we do with java failures?

**Comment by siying** [comment]
@yuslepukhin it is known that Java build is failing and @yhchiang is working on it. I think the patch is good to go.

**Comment by siying** [comment]
The change should have nothing to do with Java build.

File: utilities/write_batch_with_index/write_batch_with_index_internal.h [hunk 2]
```diff
@@ -30,7 +31,7 @@ struct WriteBatchIndexEntry {
 
   // If this flag appears in the offset, it indicates a key that is smaller
   // than any other entry for the same column family
-  static const size_t kFlagMin = std::numeric_limits<size_t>::max();
+  static const size_t kFlagMin = port::kMaxUint64;
```
Comment (siying) [inline utilities/write_batch_with_index/write_batch_with_index_internal.h]: Look at it again. Is there way to support 32-bit? Is http://stackoverflow.com/questions/27442885/syntax-error-with-stdnumeric-limitsmax related? Can we do something like that?

**Comment by yuslepukhin** [comment]
@siying It is not a problem with macro max rather with the fact that numeric_limits are not constexpr with the compiler being used.

**Comment by siying** [comment]
@yuslepukhin  it looks good. I'll merge after Travis tests pass.

**Pull request merged**